debezium-core/src/main/java/io/debezium/data/HStore.java

```diff
@@ -0,0 +1,21 @@
+package io.debezium.data;
```

Comment:
This needs the default license header (see existing files for an example). Also it should be located within the Postgres connector module, not core.

Reply:
Yes , fixed it .